render/detailed/metadata.go

```diff
@@ -0,0 +1,104 @@
+package detailed
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/weaveworks/scope/probe/docker"
+	"github.com/weaveworks/scope/probe/host"
+	"github.com/weaveworks/scope/probe/kubernetes"
+	"github.com/weaveworks/scope/probe/overlay"
+	"github.com/weaveworks/scope/probe/process"
+	"github.com/weaveworks/scope/render"
+	"github.com/weaveworks/scope/report"
+)
+
+var (
+	processNodeMetadata = renderMetadata([]MetadataRow{
+		{ID: process.PPID, Label: "Parent PID"},
+		{ID: process.Cmdline, Label: "Command"},
+		{ID: process.Threads, Label: "# Threads"},
+	})
+	containerNodeMetadata = renderMetadata([]MetadataRow{
+		{ID: docker.ContainerID, Label: "ID"},
+		{ID: docker.ImageID, Label: "Image ID"},
+		{ID: docker.ContainerState, Label: "State"},
+		{ID: docker.ContainerPorts, Label: "Ports"},
+		{ID: docker.ContainerCreated, Label: "Created"},
+		{ID: docker.ContainerCommand, Label: "Command"},
+		{ID: overlay.WeaveMACAddress, Label: "Weave MAC"},
+		{ID: overlay.WeaveDNSHostname, Label: "Weave DNS Hostname"},
+	}, getDockerLabelRows)
+	containerImageNodeMetadata = renderMetadata([]MetadataRow{
+		{ID: docker.ImageID, Label: "Image ID"},
+	}, getDockerLabelRows)
+	podNodeMetadata = renderMetadata([]MetadataRow{
+		{ID: kubernetes.PodID, Label: "ID"},
+		{ID: kubernetes.Namespace, Label: "Namespace"},
+		{ID: kubernetes.PodCreated, Label: "Created"},
+	})
+	hostNodeMetadata = renderMetadata([]MetadataRow{
+		{ID: host.HostName, Label: "Hostname"},
+		{ID: host.OS, Label: "Operating system"},
+		{ID: host.KernelVersion, Label: "Kernel version"},
+		{ID: host.Uptime, Label: "Uptime"},
+	})
+)
+
+// MetadataRow is a row for the metadata table.
+type MetadataRow struct {
```

Comment:
This type seems to be used as both a template and as instance - we might want two types for this.